package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@azure-tools/typespec-client-generator-cli": "^0.14.3"
```

Comment:
Should this `package.json` go in the root of the repo?

Reply:
I think this is a wrong package json, let me remove it

Reply:
Removed